ocs_ci/helpers/storage_agnostic_dr_helpers.py

```
@@ -0,0 +1,866 @@
+                f" --for=jsonpath='{{.status.phase}}'=Succeeded"
+                f" --timeout=60s",
+                cluster_config=cluster,
+            )
+            run_cmd(
+                f"oc delete pod mc-client -n {constants.MINIO_NAMESPACE}"
+                f" --ignore-not-found",
+                cluster_config=cluster,
+            )
+
+            run_cmd(
+                f"oc expose svc/minio -n {constants.MINIO_NAMESPACE}" f" --port=9000",
```

Ruff (0.15.17)

[warning] Implicitly concatenated string literals on one line

Combine string literals

(ISC001)